cmd/system-probe/main.go

```diff
@@ -201,18 +194,19 @@ func gracefulExit() {
 
 // versionString returns the version information filled in at build time
 func versionString(sep string) string {
-	addString := func(buf *bytes.Buffer, s, arg string, sep string) {
-		if arg != "" {
-			fmt.Fprintf(buf, s, arg, sep)
+	addString := func(buf *bytes.Buffer, format string, args ...interface{}) {
+		if len(args) > 0 && args[0] != "" {
+			_, _ = fmt.Fprintf(buf, format, args...)
 		}
 	}
 
+	av, _ := version.Agent()
+
 	var buf bytes.Buffer
-	addString(&buf, "Version: %s%s", Version, sep)
-	addString(&buf, "Git hash: %s%s", GitCommit, sep)
-	addString(&buf, "Git branch: %s%s", GitBranch, sep)
-	addString(&buf, "Build date: %s%s", BuildDate, sep)
```

Comment:
Why remove `BuildDate`? It is useful sometimes when testing multiple built versions.

Reply:
`pkg/version` doesn't have a date field, and that is the standard way version is handled in most other agents. I agree it could be useful, so maybe we could add that in a new PR?

Reply:
We could keep it as a build var for now, but not a huge concern for me.